### PR TITLE
Forms and Authentication && Advanced Forms and Active Record: Change headings sentence case

### PR DESCRIPTION
**Pull request by errantpianist**
## Because

Changed headings to sentence case in line with the Style Guide.

## This PR

- Changes headings in Forms and Authentication to sentence case
- Changes headings in Advanced Forms and Active Record to sentence case

## Issue

Related to #26169

## Additional Information

Project titles have been left capitalised as follows:
Project: Brand new project

Instructions have been written as follows: 
Step 1: do this

Please let me know if it should be another way.

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: ruby_on_rails/forms_and_authentication/project_forms.md
Comment (ManonLef) [inline ruby_on_rails/forms_and_authentication/project_forms.md:1]: The `#### Set up the Back End` should be changed to `#### Set up the back end` in this lesson :)

File: ruby_on_rails/forms_and_authentication/project_members_only.md
Comment (ManonLef) [inline ruby_on_rails/forms_and_authentication/project_members_only.md:1]: The `#### Basic Setup` header in this lesson should be changed to `#### Basic setup`

**Review by ManonLef — CHANGES_REQUESTED** [review]
Thank you so much for doing this! I left two notes on headers that still need a tweak but otherwise it looks great! Much appreciated

File: ruby_on_rails/forms_and_authentication/project_forms.md
Comment (errantpianist) [reply ruby_on_rails/forms_and_authentication/project_forms.md:1]: Thanks for spotting this! Fixed :)

File: ruby_on_rails/forms_and_authentication/project_members_only.md
Comment (errantpianist) [reply ruby_on_rails/forms_and_authentication/project_members_only.md:1]: Fixed this too :D

**Review by ManonLef — APPROVED** [review]

**Comment by ManonLef** [comment]
Thanks so much @errantpianist ! That concludes this issue.

**Pull request merged**